src/export/Export.cpp

```diff
@@ -485,6 +377,52 @@ bool Exporter::SetAutoExportOptions() {
    return true;
 }
 
+MixerOptions::Downmix* Exporter::CreateMixerSpec()
+{
+   constexpr auto MaxExportChannels = 32u;
+
+   auto exportTracks = ExportUtils::FindExportWaveTracks(TrackList::Get( *mProject ), mSelectedOnly);
+   mMixerSpec = std::make_unique<MixerOptions::Downmix>(exportTracks.size(),
+                                                        std::clamp(mPlugins[mFormat]->GetFormatInfo(mSubFormat).mMaxChannels,
+                                                                   // JKC: This is an attempt to fix a 'watching brief' issue, where the slider is
+                                                                   // sometimes not slidable.  My suspicion is that a mixer may incorrectly
+                                                                   // state the number of channels - so we assume there are always at least two.
+                                                                   // The downside is that if someone is exporting to a mono device, the dialog
+                                                                   // will allow them to output to two channels. Hmm.  We may need to revisit this.
+                                                                   // STF (April 2016): AMR (narrowband) and MP3 may export 1 channel.
+                                                                   1u,
+                                                                   MaxExportChannels));
+   return mMixerSpec.get();
+}
+
+Exporter::DownMixMode Exporter::SetUseStereoOrMonoOutput()
```

Comment:
The return value of this function is not (yet?) used at the commit that introduces the function.